src/Services/SettingsBase.cs

```diff
@@ -31,6 +37,11 @@ protected SettingsBase(IFolderData folder, IAsyncSerializer<Stream> settingSeria
             Folder = folder;
         }
 
+        /// <summary>
+        /// Gets or sets the property that determines whether to flush default values to disk.
+        /// </summary>
+        protected bool FlushDefaultValues { get; set; }
```

Comment:
This should be true by default to match existing behavior.
```suggestion
        protected bool FlushDefaultValues { get; set; } = true;
```